tests/ui/push_constant_vulkan.rs

```diff
@@ -0,0 +1,26 @@
+// Test that using push constants work.
+// NOTE(eddyb) we specifically run Vulkan validation here, as the default
+// validation rules are more lax and don't require a `Block` decoration
+// (`#[spirv(block)]` here) on `struct ShaderConstants`.
+
+// build-pass
+// compile-flags: -Ctarget-feature=+vulkan
```

Comment:
I don't think this runs vulkan validation, it only enables the vulkan memory model.

Reply:
Some we add a target feature to run Vulkan validation?